ngraph/test/backend/softmax.in.cpp

```diff
@@ -238,3 +238,29 @@ NGRAPH_TEST(${BACKEND_NAME}, softmax_overflow)
                            expf(5) / d2};
     EXPECT_TRUE(test::all_close_f(expected, read_vector<float>(result)));
 }
+
+NGRAPH_TEST(${BACKEND_NAME}, softmax_default_axis)
```

Comment:
To check default axis, you don't need to create a unit test with inference. You could do a unit test case within type_prop, create the op and use ASSERT_EQ to check on axis attribute:
`ASSERT_EQ(op->get_axis(), 1);`